packages/core-snapshots/src/contracts/database.ts

```diff
@@ -1,5 +1,5 @@
 import { Contracts } from "@arkecosystem/core-kernel";
-import { Meta, Options } from "@arkecosystem/core-snapshots/src/contracts/index";
+import { Meta, Options } from "./index";
```

Comment:
Should avoid imports from index files since they could lead to circular references in the import tree.